src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -19,7 +19,14 @@ const AuthenticatorRow = View.extend({
   className: 'authenticator-row clearfix',
   template: hbs`
     <div class="authenticator-icon-container">
-      <div class="factor-icon authenticator-icon {{iconClassName}}"></div>
+      {{#if logoUri}}
+        <div class="factor-icon authenticator-icon custom-app-logo" role="img"
```

Comment:
is this `custom-app-logo` defined in any scss file? can the style `background-image: url('{{logoUri}}')` be defined within `custom-app-logo` so the inline style can be removed?

Reply:
`logoUri` is a dynamic value which we will be getting from API response. hence adding it as inline styles dynamically. `custom-app-logo` is used in test cases to differentiate it from other `{{iconClassName}}`. Also this is Increment development, we can add more styles to this class when needed.